src/main/java/com/mycompany/exercisetwo/AnimalModel.java

```diff
@@ -0,0 +1,14 @@
+package com.mycompany.exercisetwo;
+
+/**
+ *
+ * @author Cristian Gallego
+ */
+public class AnimalModel {
```

Comment:
Excercise is requesting at least 5 parameters, you just have 4

Reply:
Corrected